File: weatherapp.client/src/app/interfaces/Current.ts
import { Condition } from "./Condition"

export interface Current {
  temp_c: number,
  is_day: number,
  condition: Condition
  wind_mph: number,
  wind_kph: number,
  wind_dir: string,
  cloud: number,
  feelslike_c: number
}

File: weatherApp.Server/Models/Current.cs
namespace weatherApp.Server.Models;

public class Current
{
    public float temp_c { get; set; }
    public int is_day { get; set; }
    public Condition condition { get; set; } = new Condition();
    public float wind_mph { get; set; }
    public float wind_kph { get; set; }
    public string wind_dir { get; set; } = string.Empty;
    public float cloud { get; set; }
    public float feelslike_c { get; set; }
}
